magento/magento2
# Fixed issue 27051: Saving an attribute with backend_type static

### Description (*)
Fixed  'frontend_class' field issue while saving an attribute with backend_type 'static'.


### Related Pull Requests


### Fixed Issues (if relevant)

1. fixes magento/magento2#27051: Saving an attribute with backend_type 'static' will remove the content of the 'frontend_class' field.

### Manual testing scenarios (*)

1. Navigate to Stores > Attributes > Product
2. Click an attribute with backend_type = static. Example: sku or category_ids
3. On the attribute edit page, you can either modify some fields or not modify any field at all.
4. Click "Save and continue edit".
5. Open Database and select **eav_attribute** table
6. Check **frontend_class** for that attribute

### Questions or comments


### Contribution checklist (*)
